src/app/FindRoute/FindRouteComponent.ts

```diff
@@ -24,7 +26,8 @@ export class FindRoute implements OnInit {
 
   constructor(private http: HttpService, private geo: UserLocationService) {}
 
-  ngOnInit() {
+  async ngOnInit() {
```

Comment:
probably don't need async if the current code and this will do the same thing where you still need to load home before routing/reports/contacts